api/all/src/main/java/io/opentelemetry/api/OpenTelemetry.java

```diff
@@ -21,6 +21,20 @@
  * @see ContextPropagators
  */
 public interface OpenTelemetry {
+  /** Returns a default, completely no-op {@link OpenTelemetry}. */
+  static OpenTelemetry getDefault() {
```

Comment:
Wondering if we should rename all these API `getDefault` to `noop` which is more obvious, and consistent with what ContextPropagators is right now.

Reply:
Though I remember there was one concern that the default tracer returns noop spans, but is itself still an operation. But I don't know if that's really an important distinction.

Reply:
I like noop. Default sounds like a default "functioning" implementation to me.

Reply:
I like "Noop", only concern is we should probably do the same everywhere...

Reply:
"Noop" is more fun to say, and certainly more precise, but I think at this point we shouldn't change it everywhere, so I think we should stick with 'default'.